src/pages/slots/SlotConfiguration.styles.ts

```diff
@@ -0,0 +1,132 @@
+import styled from "styled-components";
+import { BORDER_RADIUS, FONT_FAMILY, TERTIARY_COLOR, PRIMARY_COLOR, TEXT_COLOR, SECONDARY_COLOR } from "../../utils/Stylesheet";
+import Arrow from "../../assets/Arrow.png";
+
+
+export const ScreenContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 520px;
+    padding-top: 32px;
+    padding-bottom: 32px;
+    gap:24px;
+    border: 2px solid ${TERTIARY_COLOR};
+    border-radius: ${BORDER_RADIUS};
+`;
+
+export const MainContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    min-height: 100vh;
+    
+`
+
+export const InputContainer = styled.div`
+    display: flex;  
+    flex-direction: column;
+    gap:16px;
+`;
+
+export const EditContainer = styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 316px;
+`;
+export const Label = styled.label`
+    font-size: 16px;
+    font-family: ${FONT_FAMILY};
+   
+`;
+
+export const BaseStyle = styled.input`
+    width:300px;
+    height: 48px;
+    border-radius:${BORDER_RADIUS};
+    font-size:12px;
+    padding-left:16px;
+    color:${TEXT_COLOR};
+    font-family: ${FONT_FAMILY};
+`;
+
+export const Input = styled(BaseStyle)`
+    background-color:${SECONDARY_COLOR}; 
+    border:none;    
+    &::placeholder {
+        color: ${TEXT_COLOR};
+    }
+`;
+
+export const Select = styled(BaseStyle).attrs({ as: "select" })`
+    width:316px;
+    border:1px solid black;
+    appearance: none;      
+    webkit-appearance: none;
+    moz-appearance: none;
```

Comment:
Estos estilos deben ir con guion para que se tomen como válidos por css:    
```suggestion
    -webkit-appearance: none;
    -moz-appearance: none;
```
webkit-appearance needs to be written in camelCase as -webkit-appearance for CSS properties in styled-components.